code/modules/mob/living/simple_animal/abnormality/he/helper.dm

```diff
@@ -38,6 +38,7 @@
 		/datum/ego_datum/armor/grinder
 		)
 	gift_type =  /datum/ego_gifts/grinder
+	gift_message = "Contamination scan complete. Initiating cleaning protocol."
```

Comment:
I'd rather you not change it everywhere. The message appears when someone receives the gift, it should be as simple as "You have received the gift from X" for most abnormalities.